packages/docsearch-css/src/modal.css

```
@@ -551,6 +551,18 @@ svg.DocSearch-Hit-Select-Icon {
   width: 20px;
 }
 
+/* Hide element accessibly, so that it is still accessible to
+assistive tech users */
+.DocSearch-VisuallyHidden {
```

Maybe a more descriptive name like 
```suggestion
.DocSearch-VisuallyHiddenForAccessbility {
```

---

Thanks, updated in latest to `.DocSearch-VisuallyHiddenForAccessibility`